### PR TITLE
Redirect norwegian bank-id and fix email update

### PR DESCRIPTION
**Pull request by palmenhq**

### CODE REVIEW

**Review by oscarnylander — DISMISSED** [review]
LGTM, but I'm getting pretty bad at validating web code being good 🤷‍♂

File: src/data/graphql.tsx [hunk 1]
```diff
@@ -8294,6 +8243,8 @@ export type SignQuotesMutationFn = ApolloReactCommon.MutationFunction<
  * const [signQuotesMutation, { data, loading, error }] = useSignQuotesMutation({
  *   variables: {
  *      quoteIds: // value for 'quoteIds'
+ *      successUrl: // value for 'successUrl'
+ *      failUrl: // value for 'failUrl'
  *   },
  * });
  */
```
Comment (meletios) [inline src/data/graphql.tsx]: do we need the comments?

**Review by meletios — APPROVED** [review]
I am not a react expert but approved

File: src/data/graphql.tsx (hunk 1 above)
Comment (palmenhq) [reply src/data/graphql.tsx]: No, but this whole file is generated so I can’t help it haha 😬

**Pull request merged**